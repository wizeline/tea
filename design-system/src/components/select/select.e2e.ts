import { newE2EPage } from '@stencil/core/testing';

describe('wz-select', () => {
  it('should render a select-component', async () => {
    const page = await newE2EPage();
    await page.setContent(`
      <wz-select>
        <option selected value="1">First Option</option>
        <option option="2">Second Option</option>
      </wz-select>`);
    const select = await page.find('wz-select');
    expect(select.outerHTML).toMatchSnapshot();
  });
});
