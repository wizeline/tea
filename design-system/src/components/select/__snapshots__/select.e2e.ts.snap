// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`wz-select should render a select-component 1`] = `"<wz-select class=\\"hydrated\\"><select><option selected=\\"\\" value=\\"1\\">First Option</option><option option=\\"2\\">Second Option</option></select></wz-select>"`;
